spec/features/visitor_can_browse_items_by_categories_spec.rb

```diff
@@ -7,7 +7,8 @@
                                   description: "portrait lens",
                                   price: 12,
                                   category_id: category.id,
-                                  image_url: 'http://www.bhphotovideo.com/images/images1000x1000/sony_sel1635z_vario_tessar_t_e_16_35mm_1082052.jpg') }
+                                  image_url: 'http://www.bhphotovideo.com/images/images1000x1000/sony_sel1635z_vario_tessar_t_e_16_35mm_1082052.jpg',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.